adb/contrib/high.py

```diff
@@ -729,7 +729,7 @@ def GetLastUID(self):
 
   def GetPackages(self):
     """Returns the list of packages installed."""
-    out, _ = self.Shell('pm list packages')
+    out, _ = self.Shell('pm list packages', timeout_ms=30000)
```

Comment:
Please add a short comment as to why, e.g. "pm can be very slow at time, and a short default timeout can be confused with the device being unresponsive when in practice it's just pm that took a lot of time to return."